docs/faq.md

```diff
@@ -73,3 +73,16 @@ If you are using conventional commits in your git history, then you could swap o
 Regarding the name, [cz-js][cz-js] came first, they used the word commitizen first. When this project was created originally, the creator read "be a good commitizen", and thought it was just a cool word that made sense, and this would be a package that helps you be a good "commit citizen".
 
 [cz-js]: https://github.com/commitizen/cz-cli
+
+## How to handle revert commits?
+
+```sh
+git revert --no-commit <SHA>
+git commit -m "revert: foo bar"
+```
+
+## I got `Exception [WinError 995] The I/O operation ...` error
+
+This error is cause by a python bug on windows and it should be fixed in some newer version. (e.g. 3.8.7) So if you encounter this error. Upgrade python version may solve this.
```

Comment:
I might reword it to

```
This error was caused by a Python bug on Windows. It's been fixed and tested on Python 3.8.7. Upgrade your Python version might solve this issue.
```

Reply:
But 3.8.7 is not the only version that include this fix. According to Python's changelog, [3.8.6rc1](https://docs.python.org/3.8/whatsnew/changelog.html#python-3-8-6-release-candidate-1) and [3.9.0rc2](https://docs.python.org/3.9/whatsnew/changelog.html#python-3-9-0-release-candidate-2) should be the accurate versions first contain https://github.com/python/cpython/pull/22017.  
I am afraid that just say it has been fixed and tested on Python 3.8.7 might mislead readers this error would not happened on Python 3.9. Should I write these information to the doc?

Reply:
Yep, that would definitely be super helpful! Thanks for digging into it